src/Symfony/Component/RateLimiter/Policy/SlidingWindow.php

```diff
@@ -92,10 +84,6 @@ public function getId(): string
      */
     public function getExpirationTime(): ?int
     {
-        if ($this->cached) {
-            return null;
-        }
-
         return 2 * $this->intervalInSeconds;
```

Comment:
`return $this->windowEndAt + $this->intervalInSeconds - microtime(true);`, so that the expiration date is fixed (since this was the original intent)